### DISCUSSION TITLE
How to add logout route to NavigationItem in Panel

### DISCUSSION THREAD

**Discussion by khoait03** [filamentphp/filament#14867] [Help]
### Package

Other

### Package Version

v3.4.18

### How can we help you?

`public function panel(Panel $panel): Panel
    {
        return $panel
            ......
            ->navigationItems([
                      
                      NavigationItem::make('Logout')
                          ->url(route('filament.employer.auth.logout')) // I did wrong
                          ->sort(60)
                          ->icon('heroicon-o-arrow-left-end-on-rectangle'),

            ])
            ..................```

**Comment by leandrocfe** [comment]
What is the issue when you use this code?

**Reply by khoait03** [reply]
error: Route [filament.employer.auth.logout] not defined.

but I call the route('filament.employer.auth.logout') in the header.blade file and it works fine. I don't understand clearly whether url() can use the post route
 ```
// header.blade.php  => no error
<form action="{{ route('filament.employer.auth.logout') }}" method="POST">
    <button type="submit">Logout</button>
</form>
```

 ```
// AdminPanelProvider.php
public function panel(Panel $panel): Panel
        {
        return $panel
        ......
        ->navigationItems([

                NavigationItem::make('Logout')
                    ->url(route('filament.employer.auth.logout')) // I did wrong
                    ->sort(60)
                    ->icon('heroicon-o-arrow-left-end-on-rectangle'),

      ])
```

**Reply by leandrocfe** [reply]
`php artisan route:list --name=logout`
what is the output?

**Reply by khoait03** [reply]
POST       admin/logout ................................................................... filament.admin.auth.logout › Filament\Http › LogoutController
  POST       business/logout ............................................................. filament.employer.auth.logout › Filament\Http › LogoutController
  GET|HEAD   ung-vien/dang-xuat .......................................................... client.candidate.logout › Client\Candidate\AuthController@logout

**Reply by leandrocfe** [reply]
take a look at `POST business/logout`. This is a POST route, not GET. I think you could create a custom GET route and use `Filament::auth()->logout()` to achieve this.

**Reply by khoait03** [reply]
Thank you, I have solved the problem, in URL method even if you call GET route it gives error, only below method works

```

`` NavigationItem::make('Logout')
          ->url(config('app.url'). '/logout/business')
          ->sort(60)
          ->icon('heroicon-o-arrow-left-end-on-rectangle'),``
```

**Reply by leandrocfe** [reply]
_even if you call GET route it gives error_
What code did you try?
